src/kw_remote.sh

```diff
@@ -188,6 +182,12 @@ function add_new_remote()
     printf '  Port %s\n' "${remote_parameters['REMOTE_PORT']}"
     printf '  User %s\n' "${remote_parameters['REMOTE_USER']}"
   } >> "$remote_config_file"
+
+  # Check if user request to set this new entry a default
```

Comment:
Sorry if I'm missing the point, but there is only one default per context, so shouldn't it be '(...) new entry **_as_** default' instead of '(...) new entry **_a_** default'?